WordPress/Classes/Models/AbstractPost.m

```diff
@@ -249,7 +249,9 @@ - (void)deleteRevision
     if (self.revision) {
         [self.managedObjectContext performBlock:^{
             [self.managedObjectContext deleteObject:self.revision];
+            [self willChangeValueForKey:@"revision"];
             [self setPrimitiveValue:nil forKey:@"revision"];
+            [self didChangeValueForKey:@"revision"];
```

Comment:
You know... *THIS* was precisely the root of a [Simplenote bug which took us months to reproduce](https://github.com/Simperium/simperium-ios/issues/359).

YAY for this change!